apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -889,9 +898,9 @@ applications:
     - "0"
 machines:
   "0":
-    constraints: arch=amd64 mem=4096 root-disk=16384
+    constraints: arch=amd64 mem=8192 root-disk=40960
```

Comment:
This whole test should be deleted, ie delete TestExportBundleModelWithHardwareConstraints